### DISCUSSION TITLE
Cannot set reference value

### DISCUSSION THREAD

**Discussion by ToMdAvI** [ansys/pyfluent#2534] [Q&A]
Hello,

I would like to add a reference value to a surface (inlet) in PyFluent. However, I am struggling with the code to do this.

I currently have: 'solver.setup.reference_values.compute.from_zone_names("inlet")'

**Comment by mkundu1** [comment]
Following code will work
```
solver.setup.reference_values.compute(from_zone_name="inlet1", from_zone_type="velocity-inlet")
```
Here `compute` is a command, you can execute `help(solver.setup.reference_values.compute)` to see its list of parameters.

**Reply by ToMdAvI** [reply]
Unfortunately this didn't work.

![image](https://github.com/ansys/pyfluent/assets/153080647/cff00c9c-807d-42fb-b5e1-bbbf7aed914b)
![image](https://github.com/ansys/pyfluent/assets/153080647/591ac49b-e378-4c30-a9d2-b18695f062b2)

**Comment by mkundu1** [comment]
There seems to be an issue in Fluent GUI that "Compute from" field is not updated, but the reference values are updated for me (e.g., the reference velocity value is same as the velocity magnitude of the boundary condition).

**Reply by ToMdAvI** [reply]
Is this a problem which can be solved?

**Reply by mkundu1** [reply]
The Fluent GUI issue will be fixed in future releases of Fluent. That issue should not affect setting up or solving the simulation from PyFluent.

**Comment by ToMdAvI** [comment]
So the reference value will be set to inlet even though it isn’t stated in fluent drop-down?

**Reply by mkundu1** [reply]
That's correct.

**Reply by ToMdAvI** [reply]
Thank you :)
